src/server/github_handler.rs

```diff
@@ -735,46 +736,23 @@ impl GithubEventHandler {
                         jira::check_jira_refs(
                             &pull_request,
                             &commits,
-                            &self.config.repos().jira_projects(&self.data.repository, &branch_name),
+                            &jira_projects,
                             self.github_session.deref(),
                         );
                     }
                 }
             }
 
             // Mark JIRAs as merged
-            if is_main_branch && !is_next_branch {
-                if let Some(ref jira_config) = self.config.jira {
-                    if let Some(ref jira_session) = self.jira_session {
-                        if let Some(ref commits) = self.data.commits {
-
-                            // try to send resolve message w/ a version in it if possible
-                            let has_version =
-                                match self.config.repos().version_script(&self.data.repository, &branch_name) {
-                                    Some(_) => {
-                                        let msg = repo_version::req(
-                                            &self.data.repository,
-                                            &branch_name,
-                                            self.data.after(),
-                                            commits,
-                                        );
-                                        self.repo_version.send(msg);
-                                        true
-                                    }
-                                    None => false,
-                                };
-                            if !has_version {
-                                jira::workflow::resolve_issue(
```

Comment:
moved this versionless into `repo_version` since that was already handling that situation for the situation where we got an empty version or the version script failed.